### PR TITLE
fix badge background color (#1080)

### LINKED ISSUE #1080: [Bug] BadgeView.BackgroundColor is not changed
### Description

BadgeView.BackgroundColor is not changed after app is launched.

### Steps to Reproduce

1. Launch app which contains BadgeView
2. Change the BackgroundColor of the BadgeView

### Expected Behavior
The background color of the BadgeView is changed to the color I set.

### Actual Behavior
There was no changes on the app when I changed the BackgroundColor.
However that was applied when I changed TextColor.
xct_badge

### Basic Information

- Version with issue: https://github.com/xamarin/XamarinCommunityToolkit/commit/caa3ebc71335d68e4a69758def542b322ccdbf90
- Platform Target Frameworks: Tested on Android 10.0 emulator

### PR DESCRIPTION
**Pull request by sattew**
### Description of Change ###

Added **properyChanged** arg to "BindableProperty.Create" for a BackgroundColor.

### Bugs Fixed ###
- Fixes #1080 

### API Changes ###
None

### Behavioral Changes ###
None

### PR Checklist ###
- [ ] Has tests (if omitted, state reason in description)
- [ ] Has samples (if omitted, state reason in description)
- [x] Rebased on top of main at time of PR
- [x] Changes adhere to coding standard
- [ ] Updated [documentation](https://github.com/MicrosoftDocs/xamarin-communitytoolkit)

### CODE REVIEW

**Review by AndreiMisiukevich — APPROVED** [review]

**Comment by AndreiMisiukevich** [comment]
Nice catch :)

**Comment by AndreiMisiukevich** [comment]
@sattew can I ask you to retarget your PR to dev branch?

**Comment by sattew** [comment]
> 
> 
> @sattew can I ask you to retarget your PR to develop branch?

ofc) one moment

File: src/CommunityToolkit/Xamarin.CommunityToolkit/Views/BadgeView/BadgeView.shared.cs [hunk 1]
```diff
@@ -103,8 +103,9 @@ public bool IsAnimated
 		/// Backing BindableProperty for the <see cref="BackgroundColor"/> property.
 		/// </summary>
 		public static new BindableProperty BackgroundColorProperty =
-			BindableProperty.Create(nameof(BackgroundColor), typeof(Color), typeof(BadgeView), defaultValue: Color.Default);
-
+			BindableProperty.Create(nameof(BackgroundColor), typeof(Color), typeof(BadgeView), defaultValue: Color.Default,
+				propertyChanged: OnLayoutPropertyChanged);
```
Comment (pictos) [inline src/CommunityToolkit/Xamarin.CommunityToolkit/Views/BadgeView/BadgeView.shared.cs:107]: do we need to invalidate the entire layout?

**Comment by pictos** [comment]
@sattew 
> ofc) one moment
And can u tell me why? I wanna know that for new PR, thanks)

This is a particular moment, normally this kind of PR (bug fix) should target the `main` branch. A couple of days ago we merge a huge PR on `develop`, and to avoid any merge conflict or regression we are changing all PRs to target the `develop` branch until we merge it on `main`

File: src/CommunityToolkit/Xamarin.CommunityToolkit/Views/BadgeView/BadgeView.shared.cs (hunk 1 above)
Comment (AndreiMisiukevich) [reply src/CommunityToolkit/Xamarin.CommunityToolkit/Views/BadgeView/BadgeView.shared.cs:107]: would be better to update only the necessary part, but it's common practice for this View (other props do the same)

Comment (pictos) [reply src/CommunityToolkit/Xamarin.CommunityToolkit/Views/BadgeView/BadgeView.shared.cs:107]: I see we should back this at some point as an improvement, in another issue/PR of course. Let's ignore this right now

**Review by AndreiMisiukevich — APPROVED** [review]

**Pull request merged**